sandbox/modules/identity/src/internalClusterTest/java/org/opensearch/identity/HttpSmokeTestCaseWithIdentity.java

```diff
@@ -158,7 +158,7 @@ protected void ensureIdentityIndexIsGreen() {
         ClusterHealthResponse clusterHealthResponse = client().admin().cluster().prepareHealth().setClusterManagerNodeTimeout("1s").get();
 
         assertTrue(
-            ConfigConstants.IDENTITY_DEFAULT_CONFIG_INDEX + " index exists",
+            ConfigConstants.IDENTITY_DEFAULT_CONFIG_INDEX + " index doesn't exist",
```

Comment:
Do you mind saying why you swapped this?

Reply:
This was an oversight on my part. I wrote a message explaining what the assertion was looking for, but the message here should help a user understand why the test failed. Looks like @DarshitChanpura ran into confusion and decided to make it clearer since it was backwards.